### DISCUSSION TITLE
Modified Ordered Lists

### DISCUSSION THREAD

**Discussion by eitanlees** [quarto-dev/quarto-cli#2617] [Q&A] [answered]
In LaTeX I often use the [`enumerate`](https://ctan.org/pkg/enumerate?lang=en) package to modify ordered lists. 

```
\documentclass{article}
\usepackage{enumerate}   

\begin{document}
\begin{enumerate}[(i)]
\item First item
\item Second item
\item Third item
\end{enumerate}
\end{document}
```
<img width="189" alt="Screen Shot 2022-09-26 at 19 13 20" src="https://user-images.githubusercontent.com/2041969/192397360-3df5d149-4aaa-4b11-a26c-027f01b0d392.png">

How would this be done in Quarto?

**Comment by cderv** [comment, marked as answer]
Did you already look at the different Markdown syntax for list ? 
https://quarto.org/docs/authoring/markdown-basics.html#lists

This is from Pandoc's Markdown feature: https://pandoc.org/MANUAL.html#ordered-lists

````markdown
---
title: "Test"
format: pdf
---

This is a list 

(i) First
(i) Second
(i) Third 
````

![image](https://user-images.githubusercontent.com/6791940/192494960-0dca125b-1096-4af3-9efb-31d21a2b8f1e.png)

**Reply by eitanlees** [reply]
Thank you!
